src/App.js

```diff
@@ -1,14 +1,22 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
 import Login from "./components/login/Login";
```

Comment:
this path should be `import Login from "./components/login";`
rename login file to index.js